static/app/views/projectInstall/createProject.tsx

```diff
@@ -238,7 +235,10 @@ export function CreateProject() {
   ].filter(value => value).length;
 
   const canSubmitForm =
-    !createProject.isPending && canUserCreateProject && formErrorCount === 0;
+    !createProjectRules.isPending &&
```

Comment:
When a rule is being created, the primary button should be disabled as well because the request may fail